src/www/js/script.js

```diff
@@ -467,86 +503,100 @@ uninstallbtn.addEventListener('click', async () => {
-        Notification.show('information', 'Removing old files...');
-        fs.rm(path.join(__dirname, '..', '..', 'Server'), { recursive: true, force: true }, (err) => {
+        Notification.show('information', 'Removing old files');
+        fs.rm(path.join(__dirname, '..', '..', 'Server'), {
+            recursive: true,
+            force: true
+        }, (err) => {
             if (err) {
                 Notification.show('error', 'Failed to remove old files');
             }
         });
-        fs.rm(path.join(__dirname, '..', '..', 'Client'), { recursive: true, force: true }, (err) => {
+        fs.rm(path.join(__dirname, '..', '..', 'Client'), {
+            recursive: true,
+            force: true
+        }, (err) => {
             if (err) {
                 Notification.show('error', 'Failed to remove old files');
             }
         });
     }
     disableAll();
     // Download server files
-    Notification.show('information', 'Downloading Server files...');
+    Notification.show('information', 'Downloading Server files');
     download({
         url: 'https://osfr.editz.dev/Server.zip',
         fileName: 'Server.zip',
         temp: `${os.tmpdir()}/OSFRServer`
     }).then(() => {
         busy = true;
-        Notification.show('information', 'Installing server');
+        Notification.show('success', 'Done');
         File.extract(`${os.tmpdir()}/OSFRServer/Server.zip`, path.join(__dirname, '..', '..', 'Server'))
-        .then(() => {
-            Notification.show('success', 'Server installation complete');
-            busy = false;
-        }).catch((err) => {
-            if (err) {
-                reinstallbtn.disabled = false;
-            }
-        }).finally(() => {
-            fs.rm(path.join(os.tmpdir(), 'OSFRServer'), { recursive: true, force: true }, (err) => { });
-            busy = false;
-            serverbtn.disabled = false;
-        });
+            .then(() => {
+                Notification.show('information', 'Downloading Client Files');
+                serverbtn.disabled = true;
+            }).catch((err) => {
+                if (err) {
+                    reinstallbtn.disabled = false;
+                }
+            }).finally(() => {
+                fs.rm(path.join(os.tmpdir(), 'OSFRServer'), {
+                    recursive: true,
+                    force: true
+                }, (err) => { });
+                busy = false;
+            });
     }).catch((err) => {
         if (err) {
             Notification.show('error', err);
             reinstallbtn.disabled = false;
             uninstallbtn.disabled = false;
+            logbtn.disabled = false;
             Notification.show('error', 'Failed to download server files');
             ProgressBar.hide();
             busy = false;
         }
     }).finally(() => {
         // Download client files
         busy = true;
-        Notification.show('information', 'Downloading Client files...');

```

Comment:
Why was this notification moved?

Reply:
Its "Download Client Files" now

Reply:
Because, if i leave it there, it will first show up the client download notification then the server extracting notification (i think that looks not logically) Now it will show up as the following: Download server files > extracting server > extraction complete > downloading client files, before this it will show up like this > downloading client files > extraction complete which looks weird.

Reply:
if this is the case, then as mentioned in my other comment, perhaps we merge or rethink some of our notifications instead of moving them where they don't make sense logically.
it'd be better to accurately represent the status of the installation than make sure everything happens in the order that looks cleanest.